activation/poetdb.go

```diff
@@ -90,7 +91,7 @@ func (db *PoetDb) storeProof(proofMessage *types.PoetProofMessage) error {
 		return fmt.Errorf("could not marshal proof message: %v", err)
 	}
 
-	if err := poets.Add(db.sqlDB, ref, messageBytes, proofMessage.PoetServiceID, proofMessage.RoundID); err != nil {
+	if err := poets.Add(db.sqlDB, ref, messageBytes, proofMessage.PoetServiceID, proofMessage.RoundID); err != nil && !errors.Is(err, sql.ErrObjectExists) {
```

Comment:
if the object already exists we should return Ignore (not Accept) in p2p handler, so maybe this error needs to be handled later

Reply:
done separately in https://github.com/spacemeshos/go-spacemesh/pull/3142